File: Learning to Program/CPP/P1-Introduction/P42042.cc
#include <iostream>
using namespace std;

int main() {
    char c;
    cin >> c;
    if (c>=65 and c<=90){
        cout << "uppercase" << endl;
        if (c==65 or c==69 or c==73 or c==79 or c==85){
            cout << "vowel" << endl;
        }
        else {
            cout << "consonant" << endl;
        }
    }
    else if (c>=97 and c<=122){
        cout << "lowercase" << endl;
        if (c==97 or c==101 or c==105 or c==111 or c==117){
            cout << "vowel" << endl;
        }
        else {
            cout << "consonant" << endl;
        }
    }
}

File: Learning to Program/Lua/Introduction/P42042.lua
#!/usr/bin/env lua

vowels = "aeiou"

function inspect_letter(c)
	y = c:lower()
	print( y ~= c and "uppercase" or "lowercase")
	if (vowels:find(y) ~= nil) then
		print ("vowel")
	else print("consonant")
	end
end

c = io.read("*l")
inspect_letter(c)
